cal/cal/static/templates/categoryDetails.html

```diff
@@ -0,0 +1,7 @@
+<div>
+  <nvd3 options="categoryline" data="categoryDetails"></nvd3>
```

Comment:
super picky, but shouldn't this be "categoryLine"? just to match with your other naming conventions. be sure to change the name in app.js / wherever this ends up moving!